src/machine/board_teensy36.go

```diff
@@ -99,3 +117,17 @@ const (
 	defaultUART4RX = D34
 	defaultUART4TX = D33
 )
+
+//go:linkname sleepTicks runtime.sleepTicks
+func sleepTicks(int64)
+
+func InitPlatform() {
+	// for background about this startup delay, please see these conversations
+	// https://forum.pjrc.com/threads/36606-startup-time-(400ms)?p=113980&viewfull=1#post113980
+	// https://forum.pjrc.com/threads/31290-Teensey-3-2-Teensey-Loader-1-24-Issues?p=87273&viewfull=1#post87273
+
+	sleepTicks(25 * 1000) // 50 for TeensyDuino < 142
+	// usb_STRING_SERIAL = readSerialNumber()
+	USB0.Configure()
+	sleepTicks(275 * 1000) // 350 for TeensyDuino < 142
```

Comment:
Is there a reason not to include this directly in `postinit`? It's not like you're supposed to call `InitPlatform` (except for the runtime).

Reply:
I thought I would leave room for non-Teensy boards that are based on the NXP MK66F18. I believe these delays are more related to how the Teensy programmer detects boards than to the chip itself.

Reply:
Ah, I see. So the function is only intended to be called from the runtime but put in the machine package because it is board specific? In that case, it would be useful to add a comment explaining that, so that it's clear this function isn't supposed to be called elsewhere.